src/cpp/session/SessionSourceDatabase.cpp

```diff
@@ -60,6 +60,13 @@ namespace rstudio {
 namespace session {
 namespace source_database {
 
+// static members
```

Comment:
This is the non-trivial static object issue again. Google actually bans static objects that aren't POD entirely: http://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Static_and_Global_Variables

We can't do this, but we can try to keep string constants char* and hide static objects inside functions that return references to them.